modules/internal/ChapelBase.chpl

```diff
@@ -363,11 +363,27 @@ module ChapelBase {
       compilerError("unexpected case in exponentiation optimization");
   }
 
+  inline proc _expBaseHelp(param a: integral, b: int(?w)){ 
```

Comment:
As far as I remember, you don't need the ?w if you're not using w. (I could be wrong.)